CHANGELOG.md

```diff
@@ -3,10 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - 2021-10-31
+
+Official release of nf-core/callingcards.
+
+## v0.0.0dev - [date]
```

Comment:
?

Reply:
yes...that is a dumb changelog comment. updated in [b326514](https://github.com/nf-core/callingcards/pull/14/commits/b326514e05955ec7eb40fba59a4356d65bbd45e7)